content/teams/Codeplay

```diff
@@ -0,0 +1,81 @@
+---
+type: "team"                   
+title: DemoTeam
+name: "Codeplay"
+github-link: " https://github.com/adrikagupta "
+description: ""
+Team-Quote: ""
```

Comment:
Title would be same as `name` field.
description not given
Team-Quote not given